src/main/java/com/atlassian/bitbucket/jenkins/internal/applink/oauth/security/RSAKeys.java

```diff
@@ -0,0 +1,102 @@
+package com.atlassian.bitbucket.jenkins.internal.applink.oauth.security;
+
+import java.io.ByteArrayInputStream;
+import java.security.*;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.security.spec.EncodedKeySpec;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+import static org.apache.commons.codec.binary.Base64.decodeBase64;
+import static org.apache.commons.codec.binary.Base64.encodeBase64;
+
+public class RSAKeys {
```

Comment:
This will be used in final end to end wiring between Bitbucket Server and Jenkins